specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,2239 @@
+            "$ref": "./examples/conversations/SuccessfulConversationSummarySubmit.json"
+          },
+          "Successful Text Conversation Analysis Job Request": {
+            "$ref": "./examples/conversations/SuccessfulConversationPIISubmit.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/analyze-conversations/jobs/{jobId}": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "description": "Get the status of an analysis job. A job may consist of one or more tasks. Once all tasks are succeeded, the job will transition to the succeeded state and results will be available for each task.",
+        "operationId": "AnalyzeConversation_JobStatus",
+        "summary": "Get analysis status and results",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/JobId"
+          },
+          {
+            "$ref": "common.json#/parameters/ShowStats"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Analysis job status and metadata.",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeConversationJobState"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Conversation PII Analysis Job Status Request": {
+            "$ref": "./examples/conversations/SuccessfulConversationPIITaskStatusRequest.json"
+          },
+          "Successful Get Text Conversation Analysis Job Status Request": {
+            "$ref": "./examples/conversations/SuccessfulConversationSummarizationTaskStatusRequest.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AnalyzeConversationTaskKind": {
+      "type": "string",
+      "description": "Enumeration of supported Conversation tasks.",
+      "enum": [
+        "CustomConversation"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationTaskKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationTaskResultsKind": {
+      "type": "string",
+      "description": "Enumeration of supported conversational task results",
+      "enum": [
+        "CustomConversationResult"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationTaskResultsKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationTaskResult": {
+      "description": "The base class of a conversation task result.",
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationTaskResultsKind"
+        }
+      }
+    },
+    "CustomConversationalTask": {
```

Comment:
Despite the description saying "custom conversation task", I see this and a few other models that use "CustomConversation*al*" in the model name. Should "al" be dropped i.e., "CustomConversation{etc}"?

Reply:
CLU question. @ChongTang can you take a look?